packages/p2p/src/components/floating-rate/floating-rate.scss

```diff
@@ -118,6 +118,7 @@
     }
 
     .dc-input-wrapper {
+        // Overriding styles of input field component
```

Comment:
Is this comment necessary by the way, because its clear from the code that it is overriding the `.dc-input-wrapper`